doc/docs/master/how-tos/s3gateway.md

```diff
@@ -25,8 +25,9 @@ not work in the browser. When you use authentication, set the access and
 secret key to the same value. They are both the Pachyderm auth token used
 to issue the relevant PFS calls.
 
-If auth is not enabled on the Pachyderm cluster, use any non-empty
-credentials.
+If auth is not enabled on the Pachyderm cluster, you can use any credentials,
+even if they do not exist, empty credentials, or no
+credentials at all.
```

Comment:
I know we're following a style guide that avoids parenthesized statements, but I found this sentence _slightly_ difficult to parse `even if they do not exist` is a hedge separated by commas, but everything else is separating by commas as well.